MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -60,17 +64,17 @@ open class VisualInstructionComponent: NSObject, NSSecureCoding {
             imageURL = URL(string: "\(baseURL)@\(Int(scale))x.png")
         }
         
-        self.init(text: text, imageURL: imageURL, isDelimiter: isDelimiter)
+        self.init(text: text, imageURL: imageURL, type: type!)
     }
     
     /**
      :nodoc:
      Initialize A `VisualInstructionComponent`.
      */
-    @objc public init(text: String?, imageURL: URL?, isDelimiter: Bool) {
+    @objc public init(text: String?, imageURL: URL?, type: VisualInstructionComponentType) {
```

Comment:
It feels more natural to put `type`, a non-optional argument, ahead of `text` and `imageURL`, which may be `nil`.